plugins/sinks/http/http.go

```diff
@@ -26,6 +28,10 @@ type Config struct {
 	Headers     map[string]string `mapstructure:"headers"`
 	Method      string            `mapstructure:"method" validate:"required"`
 	SuccessCode int               `mapstructure:"success_code" default:"200"`
+	Script      *struct {
+		Engine string `mapstructure:"engine" validate:"required,oneof=tengo"`
+		Source string `mapstructure:"source" validate:"required"`
```

Comment:
this is a yaml string which has tengo script ?

Reply:
yes, similar what we have in http extractor https://github.com/goto/meteor/tree/main/plugins/extractors/http#usage

Reply:
If the field is required and mandatory, then this would be a breaking change to use with old scripts. Can we update the PR accordingly with this info?

Reply:
the script field itself is optional, verified by existing tests are not breaking,
only `script.engine` and `script.source` is required if `script` is not nil